newIDE/app/src/InstancesEditor/ViewPosition.js

```diff
@@ -6,56 +6,56 @@ import { type InstancesEditorSettings } from './InstancesEditorSettings';
 type Props = {|
   initialViewX: number,
   initialViewY: number,
-  width: number,
-  height: number,
+  initialWidth: number,
+  initialHeight: number,
   instancesEditorSettings: InstancesEditorSettings,
 |};
 
 export default class ViewPosition {
   viewX: number = 0;
   viewY: number = 0;
-  _width: number;
-  _height: number;
```

Comment:
I think that the underscore was here to signify that this property is "private" and that one should use the getWidth and getHeight methods. Maybe we should keep them although those methods do nothing but returning these attributes at the moment.

Reply:
I was confused with the fact that viewX & viewY don't have this. 
I can revert that for sure, I thought it was more readable

Reply:
reverted